esx_service/vmdk_ops.py

```diff
@@ -1523,7 +1523,7 @@ def disk_attach(vmdk_path, vm):
         if kv_status_attached:
             # KV  claims we are attached to a different VM'.
             msg += " disk {0} already attached to VM={1}".format(vmdk_path,
-                                                                 kv_uuid)
+                                                                 vm_uuid2name(kv_uuid))
```

Comment:
Does this work if the VM is not on this host?

Reply:
@shuklanirdesh82, this error message seems corrupt, its including the internal path between docker and the managed plugin. That should not be in any of the plugin messages. Not related to this issue though.

Plus if this test done with the VM attaching the volume on another host, because then vm_uuid2name() won't find the VM and it should not cause an assertion.

Reply:
@govint  by comparing the information mentioned at https://github.com/vmware/docker-volume-vsphere/issues/1702#issue-248217743 logs is improved and no more UUID and this time with correct vm name

without the proposed change:
```
docker: Error response from daemon: error while mounting volume '/var/lib/docker/plugins/07edacf1891edd3b20bd2a3cc91439b73cc449822d4df431b7c14bc5212c3cef/rootfs/mnt/vmdk/mark': VolumeDriver.Mount: Failed to add disk 'scsi1:0'. disk /vmfs/volumes/sp-sys-docker/dockvols/_DEFAULT/mark.vmdk already attached to VM=5010dd5f-26b8-3719-2032-f8a8432a106c.
```

After the proposed code change:

```
docker: Error response from daemon: error while mounting volume '/var/lib/docker/plugins/43587bbd4e6a3bdf06089bcfc677e76167baa3859c701157b6be797991a6d425/rootfs': VolumeDriver.Mount: Failed to add disk scsi1:0. disk /vmfs/volumes/datastore1/dockvols/11111111-1111-1111-1111-111111111111/createME.vmdk already attached to VM=ubuntu.
```

Reply:
Testing is updated at description (https://github.com/vmware/docker-volume-vsphere/pull/1890#issue-259055397)